webnn/concat.https.any.js

```diff
@@ -0,0 +1,370 @@
+// META: title=test WebNN API concat operation
+// META: global=window,dedicatedworker
+// META: script=./resources/utils.js
+// META: timeout=long
+
+'use strict';
+
+// https://webmachinelearning.github.io/webnn/#api-mlgraphbuilder-concat
+
+const testConcat = async (operandType, syncFlag, ins, axis, expected) => {
```

Comment:
ins?

Reply:
The **ins** is a parameter for inputs array like `[[inputData, inputShape],]`.

Reply:
I'd recommend to use more explicit name.